<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <author/>
    <comment/>
    <exportmacro/>
    <class>view::loginPage::changePassword</class>
    <widget class="QWidget" name="view::loginPage::changePassword">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>843</width>
                <height>663</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>修改密码 - LabNexus</string>
        </property>
        <widget class="QLabel" name="titleLabel">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>20</y>
                    <width>579</width>
                    <height>91</height>
                </rect>
            </property>
            <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot;
                    font-size:20pt;&quot;&gt;修改你的密码或用户名&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span
                    style=&quot; font-size:20pt;&quot;&gt;(工号不可修改)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                </string>
            </property>
        </widget>
        <widget class="QFrame" name="mainFrame">
            <property name="geometry">
                <rect>
                    <x>60</x>
                    <y>150</y>
                    <width>711</width>
                    <height>461</height>
                </rect>
            </property>
            <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
            </property>
            <widget class="QLabel" name="userIdLabel">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>20</y>
                        <width>221</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;工号:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_userId">
                <property name="geometry">
                    <rect>
                        <x>240</x>
                        <y>20</y>
                        <width>441</width>
                        <height>31</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="userNameLabel">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>70</y>
                        <width>221</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;用户名:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_userName">
                <property name="geometry">
                    <rect>
                        <x>240</x>
                        <y>70</y>
                        <width>441</width>
                        <height>31</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="oldPasswordLabel">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>120</y>
                        <width>221</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;旧密码:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_oldPassword">
                <property name="geometry">
                    <rect>
                        <x>240</x>
                        <y>120</y>
                        <width>441</width>
                        <height>31</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="newPasswordLabel">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>170</y>
                        <width>221</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;新密码:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_newPassword">
                <property name="geometry">
                    <rect>
                        <x>240</x>
                        <y>170</y>
                        <width>441</width>
                        <height>31</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="confirmPasswordLabel">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>220</y>
                        <width>221</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;再次输入密码:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_confirmPassword">
                <property name="geometry">
                    <rect>
                        <x>240</x>
                        <y>220</y>
                        <width>441</width>
                        <height>31</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="passwordHintLabel">
                <property name="geometry">
                    <rect>
                        <x>240</x>
                        <y>260</y>
                        <width>441</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot;
                        color:#ff0000;&quot;&gt;密码必须是8~16位且必须包含数字，大小写字母&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                    </string>
                </property>
            </widget>
            <widget class="QFrame" name="buttonFrame">
                <property name="geometry">
                    <rect>
                        <x>240</x>
                        <y>320</y>
                        <width>441</width>
                        <height>91</height>
                    </rect>
                </property>
                <property name="frameShape">
                    <enum>QFrame::Shape::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                    <enum>QFrame::Shadow::Raised</enum>
                </property>
                <widget class="QPushButton" name="pushButton_confirm">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>20</y>
                            <width>201</width>
                            <height>51</height>
                        </rect>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
                    </property>
                    <property name="text">
                        <string>提交申请</string>
                    </property>
                </widget>
                <widget class="QPushButton" name="pushButton_cancel">
                    <property name="geometry">
                        <rect>
                            <x>230</x>
                            <y>20</y>
                            <width>201</width>
                            <height>51</height>
                        </rect>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
                    </property>
                    <property name="text">
                        <string>取消</string>
                    </property>
                </widget>
            </widget>
        </widget>
    </widget>
    <pixmapfunction/>
    <connections/>
</ui>